packages/hub/src/app/models/[owner]/[slug]/ModelSettingsButton.tsx

```diff
@@ -17,6 +17,7 @@ export const ModelSettingsButton: FC<{
         slug
         ...MoveModelAction
         owner {
+          __typename
```

Comment:
Should be a fragment, same as in `MoveModelAction`. I was inconsistent here because I was still figuring out how Relay works and where I can cut corners, but now I think it's almost always better to use fragments.